src/main/java/org/broadinstitute/hellbender/engine/FeatureManager.java

```diff
@@ -205,18 +206,19 @@ private void initializeFeatureSources( final int featureQueryLookahead, final Co
             // Only create a data source for Feature arguments that were actually specified
             if ( featureInput != null ) {
                 final Class<? extends Feature> featureType = getFeatureTypeForFeatureInputField(featureArgument.getKey());
-                addToFeatureSources(featureQueryLookahead, featureInput, featureType, cloudPrefetchBuffer, cloudIndexPrefetchBuffer, reference);
+                addToFeatureSources(featureQueryLookahead, featureInput, featureType, cloudPrefetchBuffer, cloudIndexPrefetchBuffer, new GenomicsDBOptions(reference));
             }
         }
     }
 
-    @SuppressWarnings({"unchecked", "rawtypes"})
-    public void dumpAllFeatureCacheStats() {
```

Comment:
Maybe restore this?

Reply:
IntelliJ is telling me it's unused, but it's back regardless.